packages/vue/src/components/molecules/SfAlert/SfAlert.js

```diff
@@ -13,38 +13,29 @@ export default {
       type: String,
       default: ""
     },
-    /** One of predefined SfIcon colors, default is black.  */
-    colorIcon: {
-      type: String,
-      default: "white"
-    },
-    /** One of predefined SfIcon sizes.  */
-    sizeIcon: {
-      type: String,
-      default: "24px"
-    },
-    /** Svg file iconPath  */
-    icon: {
```

Comment:
why you decided to remove this prop?

Reply:
Because I set default param based on design. If You want to change the default icon for default type ("secondary", "info", "success", "warning", "danger") you can use slot, wdyt?